torch/_inductor/runtime/triton_heuristics.py

```diff
@@ -18,6 +18,7 @@
 from typing import Any, Dict, List, Optional, Set, Tuple
 
 import torch
+from torch._inductor import config as inductor_config
```

Comment:
The runtime heuristics here are invoked after the compilation for torchinductor has completed. so i dont know that we can directly use the inductor_config here, because patches for different modes, etc may have already applied. Thus, no use of it thus far in this file. Instead, could we just move the `os.environ.get("TORCHINDUCTOR_DUMP_LAUNCH_PARAMS", 0) == "1"` to the `__init__` of this class ?

Reply:
I see. I did think it was odd that configs weren't used here. I will move it into init.